feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableItemDetailContent.kt

```diff
@@ -168,72 +168,85 @@ private fun ArchiveSection(
-                                imageVector = Icons.Default.PlayCircle,
-                                contentDescription = null,
-                                tint = MaterialTheme.colorScheme.onPrimary,
-                                modifier = Modifier.size(width = 18.dp, height = 18.dp),
-                            )
-                            Text(
-                                text = SessionsStrings.WatchVideo.asString(),
-                                fontSize = 14.sp,
-                                color = MaterialTheme.colorScheme.onPrimary,
-                            )
-                        }
-                    },
-                    colors = AssistChipDefaults.assistChipColors(
-                        containerColor = MaterialTheme.colorScheme.primary,
-                    ),
-                    shape = RoundedCornerShape(100.dp),
-                    border = AssistChipDefaults.assistChipBorder(
-                        borderColor = MaterialTheme.colorScheme.primary,
-                    ),
-                    modifier = Modifier.align(Alignment.Center),
-                )
-            }
+            ArchiveSectionIconButton(
+                icon = {
+                    Icon(
+                        imageVector = Icons.Outlined.Description,
+                        contentDescription = null,
+                        modifier = Modifier.size(width = 18.dp, height = 18.dp),
+                    )
+                },
+                label = {
+                    Text(
+                        text = SessionsStrings.ViewDocument.asString(),
+                        fontSize = 14.sp,
+                        textAlign = TextAlign.Center,
+                    )
+                },
+                onClick = { onViewDocumentClick() },
+                modifier = Modifier
+                    .width(0.dp)
+                    .weight(1F),
+            )
+
+            ArchiveSectionIconButton(
+                icon = {
+                    Icon(
+                        imageVector = Icons.Outlined.PlayCircle,
+                        contentDescription = null,
+                        modifier = Modifier.size(width = 18.dp, height = 18.dp),
+                    )
+                },
+                label = {
+                    Text(
+                        text = SessionsStrings.WatchVideo.asString(),
+                        fontSize = 14.sp,
+                        textAlign = TextAlign.Center,
+                    )
+                },
+                onClick = { onWatchVideoClick() },
+                modifier = Modifier
+                    .width(0.dp)
+                    .weight(1F),
+            )
         }
         Spacer(modifier = Modifier.height(24.dp))
     }
 }
 
+@Composable
+private fun ArchiveSectionIconButton(
+    icon: @Composable () -> Unit,
+    label: @Composable () -> Unit,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    iconSpacing: Dp = 8.dp,
```

Comment:
Amazing! You use the Slot API pattern appropriately! Are you a Compose professional, right?

Reply:
I've always been interested in Compose, so I gave it a try.
I'm glad you liked it.

I hope your conference will be a success :)